spec/swagger_helper.rb

```diff
@@ -0,0 +1,67 @@
+# frozen_string_literal: true
+
+require "rails_helper"
+require "api/version"
+
+Dir[Rails.root.join("spec/swagger_schemas/**/*.rb")].sort.each { |f| require f }
+
+RSpec.configure do |config|
+  # Specify a root folder where Swagger JSON files are generated
+  # NOTE: If you're using the rswag-api to serve API descriptions, you'll need
+  # to ensure that it's configured to serve Swagger from the same folder
+  config.openapi_root = Rails.root.join("public/api/docs").to_s
+
+  config.openapi_strict_schema_validation = true
+
+  # Define one or more Swagger documents and provide global metadata for each one
+  # When you run the 'rswag:specs:swaggerize' rake task, the complete Swagger will
+  # be generated at the provided relative path under openapi_root
+  # By default, the operations defined in spec files are added to the first
+  # document below. You can override this behavior by adding a openapi_spec tag to the
+  # the root example_group in your specs, e.g. describe '...', openapi_spec: 'v2/swagger.json'
+  config.openapi_specs = API::Version.all.each_with_object({}) do |version, hash|
+    hash["#{version}/swagger.yaml"] = {
+      openapi: "3.0.1",
+      info: {
+        title: "NPQ Registration API",
+        version:,
+      },
+      externalDocs: {
+        description: "Find out more about Swagger",
+        url: "https://swagger.io/",
+      },
+      paths: {},
+      servers: [
+        {
+          url: "https://register-national-professional-qualifications.sandbox.education.gov.uk/",
```

Comment:
I need to look into if we can get this dynamically for the environment; I'm not sure if AKS sets an ENV var somewhere or if I'll have to stick it manually into the keyvault.

Reply:
I think we might be able to set the ENV variable here and build the url for the specific environment: https://github.com/DFE-Digital/npq-registration/blob/main/terraform/application/application.tf#L21 so we can read it in the app